android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactTVExoplayerView.java

```diff
@@ -531,15 +538,9 @@ private void doInitializePlayer(boolean force) {
 
             playerInitTime = new Date().getTime();
 
-            if (isImaStream) {
-                if (!isImaStreamLoaded && viewWidth != 0 && viewHeight != 0) {
-                    loadImaStream();
-                }
-            } else if (actionToken != null) {
-                try {
-                    player.load(source, !haveResumePosition, actionToken);
-                } catch (UnsupportedDrmException e) {
-                    handleDrmError(e);
```

Comment:
We used to `handleDrmError(e);` here. This would display a dialog on the JS layer and tell the user something went wrong when trying to load the stream. 

It doesn't look like we are making a call to `handleDrmError(e)` anymore?

Reply:
Now the UnsupportedDrmException does not throw while call load(), it may throw within onDrmSessionManagerError() while creating the DRM session.

I add the handleDrmSessionManagerError() method.